test/regress/cli/regress0/datatypes/pf-v2l60078.smt2

```diff
@@ -1,6 +1,6 @@
 ; EXPECT: unsat
 (set-logic QF_DT)
-(declare-datatypes ((n 0) (l 0) (t 0)) (((z)) ((ln) (n (r l))) ((f))))
+(declare-datatypes ((n 0) (l 0) (t 0)) (((z)) ((ln) (nc (r l))) ((f))))
```

Comment:
I assume you are doing this to allow proofs to work with Ethos? Wouldn't it make more sense to disable the tester?

Reply:
We had agreed that the "sensible" behavior is to disallow term/sort overloading, while we are still supporting it to not bother users.
In general our regressions should largely be the sensible use case unless we are specifically testing this corner case behavior.

Reply:
That sounds good. But if there isn't any `sat` regression with this kind of overloading, we should keep at least an `unsat` one to exercise that part of the parser.

Reply:
Will add.